experimental/Experimental.jl

```diff
@@ -10,6 +10,7 @@ const oldexppkgs = [
   "ModStd",
   "Rings",
   "Schemes",
+  "PlaneCurve",
```

Comment:
I will not be able to carry all the plane curve stuff over any time soon. 
This disables the experimental package. Otherwise I get a lot of warnings of identifier conflics. (see below). 
But disabling the package entirely seems drastic. @benlorenz do we have any other options? 
Keep it in a module and not export anything?
```
WARNING: using PlaneCurveModule.invert_birational_map in module Oscar conflicts with an existing identifier.
WARNING: using PlaneCurveModule.rat_normal_curve_It_Proj_Even in module Oscar conflicts with an existing identifier.
... and so on
```